packages/office-ui-fabric-react/src/components/Button/PrimaryButton/PrimaryButton.styles.ts

```diff
@@ -0,0 +1,54 @@
+import { IButtonStyles } from '../Button.Props';
+import {
+  ITheme,
+  getTheme,
+  mergeStyles,
+  mergeStyleSets
+} from '../../../Styling';
+import {
+  getStyles as getDefaultButtonStyles
+} from '../DefaultButton/DefaultButton.styles';
+
+export function getStyles(
```

Comment:
This feels like a bunch of boilerplate. theme, custom styles, palette...can we/should we abstract some of that away?